config/config.go

```diff
@@ -18,6 +18,10 @@ const (
 	DefaultDymintDir      = ".dymint"
 	DefaultConfigDirName  = "config"
 	DefaultConfigFileName = "dymint.toml"
+	MinBlockTime          = 200 * time.Millisecond
+	MaxBlockTime          = 6 * time.Second
+	MaxBatchSubmitMaxTime = 1 * time.Hour
```

Comment:
why twice `Max` ?

Reply:
I think in general we should save `max` to be system limits and without `max` to be user defined. e.g `MaxBatchSubmitTime` for the system and `BatchSubmitTime` for user. not sure if `max` makes sense in this aspect if the user is defining the param

Reply:
i renamed batchsubmittime and batchsubmitbytes params according to this.